### PR TITLE
feat(AxelarQueryAPI): add getAssetConfigFromDenom

### PR DESCRIPTION
**Pull request by codingki**
`getAssetConfigFromDenom` will improve how we get the asset information easier

### CODE REVIEW

File: src/libs/AxelarQueryAPI.ts [hunk 1]
```diff
@@ -160,4 +160,22 @@ export class AxelarQueryAPI {
     if (!assetConfig) return null;
     return assetConfig.chain_aliases[chainName].assetSymbol;
   }
+
+  /**
+   * Get the asset config for an asset on a given chain given its denom
+   * @param chainName
+   * @param denom
```
Comment (milapsheth) [inline src/libs/AxelarQueryAPI.ts:167]:
```suggestion
   * @param denom
   * @param chainName
```

**Review by milapsheth — APPROVED** [review]

File: src/libs/AxelarQueryAPI.ts [hunk 2]
```diff
@@ -160,4 +160,22 @@ export class AxelarQueryAPI {
...
+   * @param chainName
+   * @returns
+   */
+  public getAssetConfigFromDenom(denom: string, chainName: string) {
+    const allAssets = loadAssets({ environment: this.environment });
+    const assetConfig: AssetConfig | undefined = allAssets.find(
+      (assetConfig) => assetConfig.common_key[this.environment] === denom
+    );
+    if (!assetConfig) return null;
+    const result = assetConfig.chain_aliases[chainName];
```
Comment (canhtrinh) [inline src/libs/AxelarQueryAPI.ts:176]: can you null check this as well? 
to account for cases where a chainName is either not supported (or even just a typo), e.g. if someone misspells "osnosis", etc.

**Comment by canhtrinh** [comment]
thank you! i just left one small comment, but looks good otherwise

File: src/libs/AxelarQueryAPI.ts (hunk 2 above)
Comment (codingki) [reply src/libs/AxelarQueryAPI.ts:176]: Done!
https://github.com/axelarnetwork/axelarjs-sdk/pull/124/files#diff-8079341d7e2b0f653b7b3a71d36eb1172c7fc5384f2bab3df48854d3aff658ceR177

**Pull request merged**